main.go

```diff
@@ -294,8 +300,61 @@ func deploy(c *cli.Context, r *ObjectResource) error {
 	}
 }
 
+func watchStatefulSet(c *cli.Context, r *ObjectResource) error {
+	if c.Bool("debug") {
+		logDebug.Printf("sleeping 3 seconds before checking StatefulSet status for the first time")
+	}
+	time.Sleep(3 * time.Second)
```

Comment:
Should this be a constant?

Reply:
I guess, I copied it from the Deployment code... I'll update...

Reply:
pushed